### ISSUE TITLE
Error message does not fit into window

### ISSUE THREAD

**Issue by MoritzKa** [getAlby/lightning-browser-extension#812]
**Describe the bug**
The error message "not enough balance" does not fit into the notification window.

**To Reproduce**
1. go ot getalby.com/demo
2. Try to make a payment with a balance that is lower then 25 sats (+fees)
3. See error

**Expected behavior**
the text in the error message fits the into the notification window

**Information About Alby**
- 1.9.0
- installed through the browser stores 
- Wallet connected with Alby: BlueWallet

**Screenshots (If Any)**
![image](https://user-images.githubusercontent.com/13707997/164717469-14fbfca7-3518-46c9-a495-89e0322b466f.png)

**Device Information [optional]:**
 - Chrome Browser

**Additional context**
n.a.

**Are you working on this issue? (Yes/No)**
no

**Comment by escapedcat** [comment]
I believe this is the default browser alert box which can't be changed I think.  

If we're in charge of the width of the Alby extension-popup we could maybe adjust that and make it a bit wider.  
Otherwise Alby could use it's own alertbox implementation. That will be a new feature.

**Comment by bumi** [comment]
we could add a line break to the alert box.

**Comment by escapedcat** [comment]
> we could add a line break to the alert box.

Thinking outside the box, like it! :P  
Not sure if the box will be smaller because I believe there's a minimum width to these. But we should try it.

**Comment by lightningxbtc** [comment]
Hi @bumi @escapedcat 

When you use window.alert() it are uses the resources of the browser, not the web page. 

It's the browser then who is responsible for creating the alert box, and thus you cannot influence its properties (like width etc.) 

The height can be increased with `\n` but you can't control the width.

Maybe you could switch to a custom pop-up lib in react.

Currently looks like:
<img width="527" alt="Screen Shot 2022-05-04 at 17 29 14" src="https://user-images.githubusercontent.com/104893211/166655944-ebdeda39-b3b9-4a21-9342-1019b41d0e08.png">

I can change it to use something like [react-toastify](https://www.npmjs.com/package/react-toastify), quick example below.

Draft PR [here](https://github.com/getAlby/lightning-browser-extension/pull/852)

<img width="512" alt="Screen Shot 2022-05-04 at 19 25 07" src="https://user-images.githubusercontent.com/104893211/166672476-4d1a9e48-e5d9-484a-94a1-4904a5b94640.png">

However I did notice that brave also send a notification for the same error like. I don't think it's the best user experience to show the error messages in 2 places but not sure of the context here. 

<img width="375" alt="Screen Shot 2022-05-04 at 19 25 09" src="https://user-images.githubusercontent.com/104893211/166672904-05c7ab49-5ddb-4ddd-88b6-410518939e5d.png">

**Comment by escapedcat** [comment]
Brave shows the native alert plus a (brave-native?) "snackbar"?

**Comment by lightningxbtc** [comment]
> Brave shows the native alert plus a (brave-native?) "snackbar"? The toastify version looks nice. Not sure how sufficient that a genreal alert replacement (regarding longer text or even ok/cancel buttons). I guess this needs to checked from a context/UX perspective, as you said.


Yes for me on a new instance of brave by default it shows the notification toast and brave snackbar.

Alby is installed as an unpacked extension using master branch and brave is 
[Version 1.38.109 Chromium: 101.0.4951.41 (Official Build) (arm64)](https://brave.com/latest/)

<img width="1728" alt="Screen Shot 2022-05-05 at 12 36 44" src="https://user-images.githubusercontent.com/104893211/166864300-36e38f80-8754-46c3-9460-0d8704f44294.png">

From what I can see around the codebase the `alert()` for errors just use standard alerts without any ok/cancel buttons. The toast lib does support those. 

Happy to pick this up when UX decides what this should look like.
This is the lib I used [react-toastify](https://github.com/fkhadra/react-toastify) but there are many others that accomplish the same thing.
